crates/sequencer/src/commit.rs

```diff
@@ -0,0 +1,29 @@
+use blockifier::state::{
+    cached_state::CachedState,
+    state_api::{State as BlockifierState, StateReader as BlockifierStateReader},
+};
+
+pub trait Committer<S>
+where
+    for<'a> &'a mut S: BlockifierState + BlockifierStateReader,
+{
+    fn commit(cached_state: &mut CachedState<&mut S>) {
+        let diff = cached_state.to_state_diff();
+        for (address, class_hash) in diff.address_to_class_hash {
+            let _ = cached_state.state.set_class_hash_at(address, class_hash);
+        }
+        for (address, _) in diff.address_to_nonce {
+            let _ = cached_state.state.increment_nonce(address);
```

Comment:
same

Reply:
I updated for set_class_hash_at, but increment nonce should never error out.

Reply:
I think if it should never run out, it's fine to then handle the error
We never know if one day this'll change in blockifier

Reply:
fair, will update